dapps/marketplace/src/queries/Identity.js

```diff
@@ -26,6 +26,10 @@ export default gql`
           phoneVerified
           emailVerified
           websiteVerified
+          kakaoVerified
+          githubVerified
+          linkedinVerified
+          wechatVerified
```

Comment:
perhaps we should have a single field 'verified' with a parameter for type?

Reply:
What if we want to get all/multiple attestation status?

Reply:
We could use an [alias](https://graphql.org/learn/queries/#aliases), something like:
```
kakaoVerified: providerVerified(provider: "kakao")
githubVerified: providerVerified(provider: "github")
```

Just means we don't have to add more fields to the resolver every time

Reply:
Or we could have a new type `verifiedProviders` that returns an array of IDs

Reply:
I think the latter would be better as we could return other info about the attestation in the same result:

```
{
  verifiedProviders {
    id
    properties {
      type
      value
    }
  }
}
```

returns:

```
{
  verifiedProviders: [
    {
      id: 'facebook',
      properties: [
        { type: 'created', value: 'Jan 1, 2016' },
        { type: 'friends', value: '240' },
      ]
    },
    {
      id: 'twitter',
      properties: [
        { type: 'created', value: 'Feb 4, 2017' },
        { type: 'followers', value: '600' },
      ]
    },
    {
      id: 'website',
      properties: [
        { type: 'url', value: 'https://poulden.com' },
        { type: 'created', value: 'Mar 7, 2011' },
      ]
    }
  ]
}
```

Maybe we can do this in a separate PR though.

Reply:
Sure. Will open another PR for that.